src/components/ParamsAccordion.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+  FormHelperText,
+  Typography,
+  MenuItem
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { InputParams } from "@/models/workflow";
+import FTextField from "@/components/form/FTextField";
+import DragDropUploader from "./DragDropUploader";
+
+interface ParamAccordionGroupProps {
+  groupKey: string;
+  groupParams: InputParams[];
+}
+
+export default function ParamAccordionGroup({
```

Comment:
This is a nicely done dynamic form! Always tricky to put those together so it's nice to see